### ISSUE TITLE
CocoaPods

### ISSUE THREAD

**Issue by lfarah** [FlexMonkey/Blurable#5]
I'd be cool to also add Cocoapods support. I done this implementation myself last month for my library. Just follow [this post](https://guides.cocoapods.org/making/making-a-cocoapod.html) and [this checklist](https://github.com/thellimist/EZPods).

**Comment by FlexMonkey** [comment]
Even with SPM on the way?

:)

Simon

**Comment by lfarah** [comment]
@FlexMonkey Yeah, definitely. Many apps use CocoaPods a lot! And it will only take you 10 minutes to set it up!

**Comment by lfarah** [comment]
@FlexMonkey can we reopen this?
